### PR TITLE
(feat) Add ninja img template

### PR DESCRIPTION
**Pull request by PascalPiche**

### CODE REVIEW

File: workspace/utilities/lib/ninja-img.xsl [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		</xsl:call-template>
+	</xsl:template>
+
+	<xsl:template match="//@*" mode="ninja-img">
+		<xsl:attribute name="{name(.)}">
+			<xsl:value-of select="."/>
+		</xsl:attribute>
+	</xsl:template>
+
+</xsl:stylesheet>
```
Comment (fhamon) [inline workspace/utilities/lib/ninja-img.xsl]: Missing an empty line 😁

File: workspace/utilities/lib/ninja-img.xsl [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		<xsl:variable name="src" select="@src" />
+		<xsl:variable name="is-gif" select="contains($src, '.gif')" />
+		<xsl:variable name="is-svg" select="contains($src, '.svg')" />
+		<xsl:variable name="is-local" select="contains($src, $root)" />
+		<xsl:choose>
+			<xsl:when test="$is-local and not($is-gif) and not($is-svg)">
+				
+				<xsl:variable name="new-path" select="substring-after($src, 'workspace/')" />
+
+				<xsl:element name="{name()}">
```
Comment (fhamon) [inline workspace/utilities/lib/ninja-img.xsl]: We should call the image xslt template if we can

File: workspace/utilities/lib/ninja-img.xsl [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+		</xsl:element>
+	</xsl:template>
+
+	<xsl:template match="img" mode="ninja-img">
+		<xsl:variable name="src" select="@src" />
+		<xsl:variable name="is-gif" select="contains($src, '.gif')" />
+		<xsl:variable name="is-svg" select="contains($src, '.svg')" />
+		<xsl:variable name="is-local" select="contains($src, $root)" />
+		<xsl:choose>
+			<xsl:when test="$is-local and not($is-gif) and not($is-svg)">
```
Comment (nitriques) [inline workspace/utilities/lib/ninja-img.xsl]: Add tiff ??

**Comment by dominic-blain** [comment]
C'est utilisé pour quoi? Et ça fonctionne avec le nouveau picture.xsl de @fhamon ?

File: workspace/utilities/lib/ninja-img.xsl (hunk 1 above)
Comment (PascalPiche) [reply workspace/utilities/lib/ninja-img.xsl]: fixed in 2b782c6718bf38acdbd84dd0ee3179b0a84ad787

File: workspace/utilities/lib/ninja-img.xsl (hunk 2 above)
Comment (PascalPiche) [reply workspace/utilities/lib/ninja-img.xsl]: not sure for that... why?

File: workspace/utilities/lib/ninja-img.xsl (hunk 3 above)
Comment (PascalPiche) [reply workspace/utilities/lib/ninja-img.xsl]: Fixed in e7b6aa5f00eb59dab8285095fe15bbc51c09d6b1

**Comment by PascalPiche** [comment]
@dominic-blain: Its for all image injected in the markdown for activating the jit process on the width of the picture.

**Pull request merged**